frontend/src/views/web/change-password.html

```diff
@@ -4,27 +4,35 @@
-                            <div class="wrn-msg text-highlight" ng-show="changepwd.isValid.old_password">{{changepwd.wrnMsg.old_password}}</div>
+                            <div class="wrn-msg text-highlight" ng-show="changepwd.isValid.old_password" ng-messages="changepwdForm.old_password.$error">{{changepwd.wrnMsg.old_password}}
+                                <p ng-message="required">Old password is required</p>
+                            </div>
                         </div>
 
                         <div class="input-field align-left">
-                            <input id="new_password1" type="password" class="validate text-dark-black dark-autofill w-400" ng-model="changepwd.user.new_password1" focus-if>
+                            <input id="new_password1" type="password" class="validate text-dark-black dark-autofill w-400" ng-model="changepwd.user.new_password1" ng-minlength="8" required focus-if>
                             <span class="form-icon form-icon-dark"><i class="fa fa-user"></i></span>
                             <label for="new_password1" data-error="wrong" data-success="right">New Password*</label>
-                            <div class="wrn-msg text-highlight" ng-show="changepwd.isValid.new_password1">{{changepwd.wrnMsg.new_password1}}</div>
+                            <div class="wrn-msg text-highlight" ng-show="changepwd.isValid.new_password1" ng-messages="changepwdForm.new_password1.$error">{{changepwd.wrnMsg.new_password1}}
+                                <p ng-message="minlength">password is less than 8 characters.</p>
+                                <p ng-message="required">New password is required</p>
+                            </div>
                         </div>
 
                         <div class="input-field align-left">
-                            <input id="new_password2" type="password" class="validate text-dark-black dark-autofill w-400" ng-model="changepwd.user.new_password2" focus-if>
+                            <input id="new_password2" type="password" class="validate text-dark-black dark-autofill w-400" ng-model="changepwd.user.new_password2" ng-minlength="8" compare-to="changepwd.user.new_password1" required focus-if>
                             <span class="form-icon form-icon-dark"><i class="fa fa-user"></i></span>
                             <label for="new_password2" data-error="wrong" data-success="right">New Password (Again)*</label>
-                            <div class="wrn-msg text-highlight" ng-show="changepwd.isValid.new_password2">{{changepwd.wrnMsg.new_password2}}</div>
+                            <div class="wrn-msg text-highlight" ng-show="changepwd.isValid.new_password2" ng-messages="changepwdForm.new_password2.$error">{{changepwd.wrnMsg.new_password2}}
+                                <p ng-message="minlength">password is less than 8 characters.</p>
+                                <p ng-message="required">Confirm new password is required</p>
```

Comment:
Add ng-message for `compareTo` saying that `Password do not match`. See the reference in `signup/reset password confirm`.